sdks/go/pkg/beam/core/runtime/exec/dynsplit_test.go

```diff
@@ -103,11 +103,11 @@ func TestDynamicSplit(t *testing.T) {
 			// with the input coder to the path.
 			// TODO(BEAM-10579) Switch to using splittable unit's input coder
 			// once that is implemented.
-			p, err := decodeDynSplitElm(splitRes.split.PS, cdr)
+			p, err := decodeDynSplitElm(splitRes.split.PS[0], cdr)
```

Comment:
Is there a convention here you could use to assert that there's only one element while getting it, rather than let any (unexpected?) other elements in the list be dropped?

Reply:
That would be useful. @lostluck , know of anything? If not I can just explicitly check that it only has a length of one, because that is an expectation of the test.

Reply:
Go, being a simple language would have you put the if in there. It would be a waste to wrap it in a function unless the pattern is repeating. At least not until generics land and we can define a `GetOnlyElement[T any](slice []T) { ... }` helper.